bin/console

```diff
@@ -1,14 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
This is a shortcut to open `irb` with the gem included (`rails console` equivalent for the gem). It's ok to be in the `bin` directory as it's a development binary, and it won't be pushed to the gem because it's not included in the gemspec's `executables` attribute.